qnm_filter/utility.py

```diff
@@ -0,0 +1,79 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+"""
+Created on Mon Feb 20 17:55:14 2023
+
+@author: neil
+"""
+
```

Comment:
Remove the lines above. I will add you as an author somewhere else. Also, add a one-line comment and `__all__` to the top of the file. See other modules for example.